regolith/helpers/l_membershelper.py

```diff
@@ -0,0 +1,90 @@
+"""Helper for listing group members.
+
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.dates import get_due_date
+from regolith.helpers.basehelper import SoutHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "people"
+HELPER_TARGET = "l_members"
+ALLOWED_STATI = ["proposed", "started", "finished", "back_burner", "paused", "cancelled"]
+
+
+
+def subparser(subpi):
+    subpi.add_argument("-v", "--verbose", action="store_true", help='increase verbosity of output')
+    subpi.add_argument("-l", "--lead",
```

Comment:
remove all unused args.  I think -l -p and -s should be removed but please check they are not currently used first.